src/runtime_src/core/edge/common/aie_parser.cpp

```diff
@@ -251,6 +277,18 @@ get_gmios(const xrt_core::device* device)
   return ::get_gmio(aie_meta);
 }
 
+std::vector<plio_type>
+get_plios(const xrt_core::device* device)
+{
+  auto data = device->get_axlf_section(AIE_METADATA);
+  if (!data.first || !data.second)
+    return std::vector<plio_type>();
```

Comment:
`return {} `
should do it.

Reply:
Let's leave this case as it for now because all other functions like get_rtp, get_gmios, get_tiles etc. are using the same style. We can improve it in a later code clean up.